### PR TITLE
fix(condo): DOMA-7749 get cookie_keys for oidc provider from env

### PR DESCRIPTION
**Pull request by sitozzz**
This PR fix security issue with OIDC provider

- [x] Secret values updated at helm submodule

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,6 +3,7 @@ REDIS_URL=redis://127.0.0.1:6379
 NODE_ENV=development
 DISABLE_LOGGING=true
 COOKIE_SECRET=random
+COOKIE_KEYS='["OEK5zjaAMPc3L6iK7PyUjCOziUH3rsrMKB9u8H07La1SkfwtuBoDnHaaPCkG5Brg", "MNKeIebviQnCPo38ufHcSfw3FFv8EtnAe1xE02xkN1wkCV1B2z126U44yk2BQVK7"]'
```
Comment (SavelevMatthew) [inline .env.example:6]: Maybe add OIDC_ prefix?

File: apps/condo/domains/user/oidc/configuration.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ const HTTPS_REGEXP = /^https:/
 
 module.exports = function createConfiguration (context, conf) {
     const jwksStr = get(conf, 'JWKS')
+    const cookieKeys = JSON.parse(get(conf, 'COOKIE_KEYS', '[]'))
+
+    if (isEmpty(cookieKeys)) {
```
Comment (Alllex202) [inline apps/condo/domains/user/oidc/configuration.js:13]: Do we need to check that it is an array?

File: .env.example (hunk 1 above)
Comment (sitozzz) [reply .env.example]: Fixed

File: apps/condo/domains/user/oidc/configuration.js (hunk 2 above)
Comment (sitozzz) [reply apps/condo/domains/user/oidc/configuration.js]: Done

**Review by SavelevMatthew — APPROVED** [review]

**Pull request merged**